src/mlx90632.c

```diff
@@ -320,7 +320,7 @@ static double mlx90632_calc_temp_object_iteration(double prev_object_temp, int32
  *                              input 25.0
  * @param[in] object object temperature from @link mlx90632_preprocess_temp_object @endlink
  * @param[in] TAdut ambient temperature coefficient
- * @param[in] Tr reflected (environment) temperature from a sensor different than the MLX90632 or acquired by other means
+ * @param[in] TaTr4 compensation coefficient for reflected (environment) temperature
```

Comment:
Previous was more descriptive for developers, so maybe we combine them together?

Reply:
Tr is indeed the reflected temperature, but TaTr4 is a different coefficient that is calculated based on the Ta, Tr and the emissivity (and is to the power of 4)

Reply:
```suggestion
 * @param[in] TaTr4 compensation coefficient for reflected (environment) temperature. The compensation 
                       coefficient is calculated from ambient temperature from a sensor different than the MLX90632 or
                       acquired by other means multiplied by objects reflected temperature obtained by sensor to power of 4
```